refs/files-backend.c

```diff
@@ -3298,6 +3298,11 @@ static int files_init_db(struct ref_store *ref_store,
 	return 0;
 }
 
+static int files_fsck_refs(struct ref_store *ref_store)
+{
+	return 0;
```

Comment:
I think it's great that this commit doesn't yet add any checks. But to demonstrate how this is expected to be used you should add a second commit to this patch series that introduces some simple check so that we can write a test for this new logic.